### PR TITLE
Ien 434 basic extract custom date

### PR DESCRIPTION
**Pull request by coryatwater-FW**
need to find something for the formatting and add a validator for empty dates, but functionality is there

### CODE REVIEW

**Review by sunghwan-park-fw — COMMENTED** [review]
Looks good. just one comment.

File: apps/web/src/components/reporting/DataExtractReport.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  return (
+    <>
+      <div className='text-bcBluePrimary text-lg font-bold mb-4'>Data Extract</div>
+      <Formik initialValues={initialValues} onSubmit={download}>
+        {({ isSubmitting, handleReset, dirty, touched }) => (
+          <FormikForm>
+            <div className='flex flex-row pb-4'>
+              <span className='pr-3 w-full'>
+                <Field name='from' label='Start Date' type='date' max={getMaxDate()} />
```
[the remark this answers is not in the document]
Comment (coryatwater-FW) [reply apps/web/src/components/reporting/DataExtractReport.tsx:40]: yes, will fix that

File: apps/web/src/components/reporting/DataExtractReport.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <div className='text-bcBluePrimary text-lg font-bold mb-4'>Data Extract</div>
+      <Formik initialValues={initialValues} onSubmit={download}>
+        {({ isSubmitting, handleReset, dirty, touched, values }) => (
+          <FormikForm>
+            <div className='flex flex-row pb-4'>
+              <span className='pr-3 w-full'>
+                <Field name='from' label='Start Date' type='date' max={values.to || getMaxDate()} />
+              </span>
+              <span className='pr-3 w-full'>
+                <Field name='to' label='End Date' type='date' max={getMaxDate()} />
```
Comment (sunghwan-park-fw) [inline apps/web/src/components/reporting/DataExtractReport.tsx:43]: it just came to my mind. what about the min of end date?

**Comment by sunghwan-park-fw** [comment]
fyi, I fixed the cypress test

**Review by sunghwan-park-fw — APPROVED** [review]

**Pull request merged**